spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -7,62 +7,156 @@
 
   describe 'GET#index' do
     context 'given a logged in user' do
-      context 'given the user has keywords' do
-        it 'returns success status' do
-          user = Fabricate(:user)
-          Fabricate.times(3, :keyword, user: user)
+      context 'given the user retrives all of their keywords' do
+        context 'given the user has keywords' do
+          it 'returns success status' do
+            user = Fabricate(:user)
+            Fabricate.times(3, :keyword, user: user)
 
-          get api_v1_keywords_path, headers: create_token_header(user)
+            get api_v1_keywords_path, headers: create_token_header(user)
 
-          expect(response).to have_http_status(:success)
-        end
+            expect(response).to have_http_status(:success)
+          end
+
+          it 'returns three items' do
+            user = Fabricate(:user)
+            Fabricate.times(3, :keyword, user: user)
 
-        it 'returns three items' do
-          user = Fabricate(:user)
-          Fabricate.times(3, :keyword, user: user)
+            get api_v1_keywords_path, headers: create_token_header(user)
 
-          get api_v1_keywords_path, headers: create_token_header(user)
+            response_body = JSON.parse(response.body, symbolize_names: true)
+            expect(response_body[:data].count).to eq 3
+          end
 
-          response_body = JSON.parse(response.body, symbolize_names: true)
-          expect(response_body[:data].count).to eq 3
+          it 'returns metadata with page = 1, per_page = 2 and total_items = 3' do
+            user = Fabricate(:user)
+            Fabricate.times(3, :keyword, user: user)
+
+            get api_v1_keywords_path, params: { page: 1, per_page: 2 }, headers: create_token_header(user)
+            response_body = JSON.parse(response.body, symbolize_names: true)
+
+            expect(response_body[:meta]).to eq(page: 1, per_page: 2, total_items: 3)
+          end
         end
 
-        it 'returns metadata with page = 1, per_page = 2 and total_items = 3' do
-          user = Fabricate(:user)
-          Fabricate.times(3, :keyword, user: user)
+        context 'given the user has no keyword' do
+          it 'returns success status' do
+            Fabricate(:keyword)
+
+            get api_v1_keywords_path, headers: create_token_header
+
+            expect(response).to have_http_status(:success)
+          end
+
+          it 'returns an empty array' do
+            Fabricate(:keyword)
+
+            get api_v1_keywords_path, headers: create_token_header
+
+            response_body = JSON.parse(response.body, symbolize_names: true)
+            expect(response_body[:data].count).to eq 0
```

Comment:
```suggestion
            expect(response_body[:data]).to eq []
```

or 

```suggestion
            expect(response_body[:data]).to be_empty
```

In that second case, we must change the description to "it is empty" because an empty result can be any type of collection, not only an array :)